src/datoms_differ/datom.cljc

```diff
@@ -116,39 +115,34 @@
                c#)))
          res))))
 
+(defprotocol DatomValueComparator
+  "A protocol that allows you to provide custom compare for types of your choosing for the value attribute of a datom"
+  (customCompareTo [this other]))
```

Comment:
What's up with camelCasing, bro? :)

I'd also rename this function to "compare-value". The "custom" part is implicit in this being an protocol and thus extensible.

Reply:
👍